clients/benchmarks/CMakeLists.txt

```
@@ -81,8 +81,11 @@ if (NOT WIN32)
     target_link_libraries( hipblaslt-bench PRIVATE lapack cblas )
   endif()
   list( APPEND COMMON_LINK_LIBS "-lm -lstdc++fs")
-  if (NOT BUILD_FORTRAN_CLIENTS)
-    list( APPEND COMMON_LINK_LIBS "-lgfortran -lflang -lflangrti") # for lapack
+
+  if (CMAKE_Fortran_COMPILER_ID MATCHES "GNU")
+    list( APPEND COMMON_LINK_LIBS "-lgfortran") # for lapack
+  else()
+    list( APPEND COMMON_LINK_LIBS "-lflang -lflangrti") # for lapack
   endif()
```

Will the following not work?

```suggestion
   find_package(LAPACK Required)
   target_link_libraries(hipblaslt-bench PRIVATE LAPACK::LAPACK)
```

---

The cmake config varies so much it wasn't reliable in the past, nor do we want to add an entire file of config rules just to find it.  This was meant as a quick fix but is dragging on due to other CI failures.   Could try to move to a find_package fallback (as AOCL will become default) but I think we will ask @NaveenElumalaiAMD to remove the lapack dependency entirely like done in rocblas.

---

I think Eiden you should become a code owner, and so only 1 team member review required?